### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by AminaBuhari**
I collaborated with my partner @RaoAkif and created a contact form in my portfolio project. We updated:
1. The index.html file and added a contact section.
2. The style.css according to the figma template given to us.
3. We added a free form link which allows contact forms filled to be redirected to the email provided.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -303,6 +301,50 @@ <h5 class="header-5">About Me</h5>
...
         </ul>
     </section>
-</body>
-
+    <section id="contact-form">
+        <div id="header-6">
+            <h3>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h3>
+        </div>
```
Comment (Donard97) [inline index.html:309]: - [ ] You did an amazing job add contact form section but kindly change it to footer tag as it was part of [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). Keep rocking 💪

File: index.html [hunk 2]
```diff
@@ -303,6 +301,50 @@ <h5 class="header-5">About Me</h5>
...
+        <div>
+            <form action="https://formspree.io/f/xvolakyp" method="POST">
+                <input class="form-input" type="text" name="fullname" placeholder="Full name" required maxlength="30" />
+                <input class="form-input" type="email" name="email" placeholder="Email address" required />
+                <textarea id="text-area" name="message" required maxlength="500">
+Enter text here</textarea
+          >
+          <button class="button-3h" type="submit">Get in touch</button>
+        </form>
```
Comment (Donard97) [inline index.html:318]: - [ ] Good job on adding button but please stick as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) and kindly add button interactions as it is shown on design:

image

File: index.html [hunk 3]
```diff
@@ -303,6 +301,50 @@ <h5 class="header-5">About Me</h5>
             </li>
         </ul>
     </section>
-</body>
-
+    <section id="contact-form">
+        <div id="header-6">
+            <h3>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h3>
```
Comment (Donard97) [inline index.html:308]: - [ ] Please change font of the heading following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468). Keep coding 💪

**Review by Donard97 — CHANGES_REQUESTED** [review]
## Hi @AminaBuhari , 👋 

_You did an amazing job so far! 👏 💯 💯_ 

There are some issues that you still need to work on to go to the next project but you are almost there! 👍 

giphy (4)

## Required Changes ♻️

- [ ] Please check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by gemmen29 — APPROVED** [review]
Hi @AminaBuhari, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**